README.md

```diff
@@ -1,11 +1,11 @@
 dicthash
 ========
```

Comment:
Nice work. What about adding a python blob for 2.7 and 3.5 in addition?
You can check the README of the h5py_wrapper for example code. The python 3.5 blob can be linked similarly, like so: https://img.shields.io/badge/python-3.5-blue.svg